src/components/quotes.scss

```diff
@@ -0,0 +1,175 @@
+.quotes-container {
+  width: 100%;
+  margin-top: var(--sectionSpacingPrimary);
+  display: flex;
+  justify-content: center;
+  position: relative;
+}
+
+.quotes-container__background {
+  background-color: #f1efec;
+  width: 80%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+}
+
+.quotes-container__wrapper {
+  max-width: calc(1440px + 2 * 84px);
```

Comment:
Do we already have 1440px saved as a max-width variable? I know it powers the overall page width, so let's go ahead and set that up if not.

Reply:
Added that in! I also added the max-width calculation since it is repeated throughout the site.